generator/inceptionmain.go

```diff
@@ -234,6 +241,7 @@ func (im *InceptionMain) Run() error {
 
 		os.Remove(im.TempMainPath)
 		os.Remove(im.exposePath)
+		os.Remove(im.tempDir)
```

Comment:
Sure, we can do that. I don't think there should be anything else in the directory except the file removed on line 242, but it can't hurt.